sources/packages/api/src/types/institutionLocation.ts

```diff
@@ -0,0 +1,14 @@
+export interface ValidatedInstitutionLocation {
+  data: {
+    locationName: string,
+    address1: string,
+    address2?: string,
+    city: string,
+    provinceState: string,
+    postalZipCode: string,
+    country: string,
+    applicationId: string,
```

Comment:
Is there any particular reason these properties be present here?
- applicationId: string,
- pid: string,
- process_pid: string